cpp/map_module/algorithm/map.cpp

```diff
@@ -1 +1,86 @@
 #include "map.hpp"
+#include <string>
+#include "mg_utils.hpp"
+#include "mgp.hpp"
+
+namespace {
+std::string value_to_string(const mgp::Value &value) {
+  switch (value.Type()) {
+    case mgp::Type::Null:
+      return "";
+    case mgp::Type::Any:
+      return "";
+    case mgp::Type::Bool:
+      return (value.ValueBool() ? "true" : "false");
+    case mgp::Type::Int:
+      return std::to_string(value.ValueInt());
+    case mgp::Type::Double:
+      return std::to_string(value.ValueDouble());
+    case mgp::Type::String:
+      return std::string(value.ValueString());
+    case mgp::Type::List:
+      return "";
+    case mgp::Type::Map:
+      return "";
```

Comment:
Can we support that via recursive calls? I think it makes sense to add that as part of mgp::Value and maybe overload operator << for mgp::Value directly. What do you think?